### ISSUE TITLE
WithTracking feature does not appear to work

### ISSUE THREAD

**Issue by rhuynh** [marcj/css-element-queries#61]
On the latest code on main, when using ElementQueries.withTracking = true, the instance method detach() doesn't detach all the elements that it's supposed to track because the tracked elements array is empty.

Steps to reproduce:
1.  set ElementQueries.withTracking = true;
2.  call ElementQueries.init();
3.  at a later point, call ElementQueries.instance.detach();
4.  Notice that in detach() the elements[] is empty (line 274)

Expected behavior:
When configured withTracking = true, calling ElementQueries.instance.detach() should remove all tracked elements.
